package.json

```diff
@@ -1,32 +1,44 @@
 {
   "name": "@hubspot/prettier-plugin-hubl",
-  "version": "0.2.8",
-  "main": "prettier-plugin-hubl.js",
+  "version": "0.3.0",
+  "main": "./prettier/dist/src/index.js",
+  "private": true,
```

Comment:
Yes, sort of. I forgot to fix this. There's a new commit [2c3ef97](https://github.com/HubSpot/prettier-plugin-hubl/pull/76/commits/2c3ef976b6090ca5c57431c795dc9595f1bc081e) that contains the changes I'm describing here, but:

- We use Yarn workspaces, which lets Yarn manage the monorepo structure. Yarn workspaces [requires that the "root" package be marked as private](https://classic.yarnpkg.com/lang/en/docs/workspaces/#:~:text=Note%20that%20the%20private%3A%20true%20is%20required!), because you don't want to publish your entire workspace. If you try to use Workspaces with a public root package, Yarn errors. So yes, this package.json should be marked as private.
- I overlooked the fact that this means that we need to change which part of the project we're actually publishing. The monorepo setup only really makes sense if we're publishing individual packages, not if we want to publish the entire thing (which is why I don't think this should be a monorepo in the first place, but anyways). 
- So the change made in the commit linked above is to make the `prettier` package the thing we actually publish. This is done by: 
    - Renaming the root package json from `@hubspot/prettier-plugin-hubl` to `root` (the name doesn't matter, really).
    - Renaming the `prettier` package from `prettier-printer-hubl` to `@hubspot/prettier-plugin-hubl` and changing the version from `1.1.0` to `0.3.0` (which is the next pubic NPM version)
 
 
 This means that Yarn linking for local dev becomes a little bit different, you need to `yarn link` in the root of the `prettier` folder. 
 
So this is actually a pretty major change in project structure. Sorry about that.

Reply:
Yeah, I see previously the package was just pointing to the `prettier-plugin-hubl.js` file which was basically just importing the `prettier/` dist -- so the new commit addition makes sense.